app/views/admin/_admin_options.html.erb

```diff
@@ -1,16 +1,42 @@
 <!-- BEGIN admin options -->
-  <div class="admin">
-	<h3 class="landmark heading">Admin Actions</h3>
+<div class="admin">
+  <% item_type_name = item.class == ExternalWork ? "External Work" :
```

Comment:
I suppose in that fabled "ideal world", we'd add a localisable `display_name` property to all the relevant classes so we could just call that whenever we need the name of the thing. But your solution sounds like a good way to solve it without fiddling with the classes themselves.